react/features/notifications/components/web/NotificationsContainer.js

```diff
@@ -166,12 +163,9 @@ class NotificationsContainer extends Component<Props> {
             <AtlasKitThemeProvider mode = 'light'>
                 <FlagGroupContext.Provider value = { this._api }>
                     <div
-                        className = { `${this.props.classes.container} ${this.props.portal
-                            ? this.props.classes.containerPortal
-                            : this.props._isChatOpen
-                                ? this.props.classes.containerChatOpen
-                                : ''}`
-                        }
+                        className = { clsx(this.props.classes.container, {
+                            [this.props.classes.containerPortal]: this.props.portal
```

Comment:
As I recall, you have started using `clsx` recently. `clsx` is a simple library and it is used in many frontend projects. I don't believe using `clsx` would cause any problem for other developers. But again, if you think that it should be changed, I change it.

Reply:
I mean probably using objects does not confuse other people. Because almost everyone use it ,explained in the README file, and its source code is less than 50 lines of code.

Reply:
I understand, but I think that the  `&&` notation is understandable even to those that haven't read the docs. And because I don't see any major benefit to using the object notation instead of the `&&` notation I'd rather we use the simplest one. Considering this is an open-source project we'd like to keep it simple so that even new programmers can contribute. Thanks for understanding